tools/ansible/wsu.yaml

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
Please work with @suhanime on the layout of the ansible playbooks. cc: @ravisantoshgudimetla